home/src/main/resources/rdf/i18n/de_DE/interface-i18n/firsttime/vitro_UiLabel.ttl

```diff
@@ -6445,3 +6445,28 @@ uil-data:password_reset_admin_notification_email_html.Vitro
         uil:hasApp       "Vitro" ;
         uil:hasKey       "password_reset_admin_notification_email_html" ;
         uil:hasPackage   "Vitro-languages" .
+
+uil-data:browse_results_alphabetical_index.Vitro
+        rdf:type         owl:NamedIndividual ;
+        rdf:type         uil:UILabel ;
+        rdfs:label       "a,b,c,d,e,f,g,h,i,j,k,l,m,n,o,p,q,r,s,t,u,v,w,x,y,z"@de-DE ;
```

Comment:
should be extended with German letters. This might be a task for a native German speaker, but it looks to me this might be ä, ö, ü, ß at the end after z - [link](https://www.reddit.com/r/NoStupidQuestions/comments/z7ye4a/how_does_the_german_alphabetic_order_work/#:~:text=So%2C%20the%20alphabet%20goes%20from,not%20one%20after%20the%20other.)

Reply:
DIN 5007 should be applied for sorting- https://de.wikipedia.org/wiki/Alphabetische_Sortierung#Deutschland 

Additional letter for Umlauts in the browsing list are not necessary.